website/components/layouts/Header/Header.tsx

```diff
@@ -80,6 +153,16 @@ export function NavPopover({
                 GitHub
               </a>
             </li>
+            <li>
+              <a
+                href="https://valtio-xnj9.vercel.app/zh"
```

Comment:
This link will need to change to something like `http://zh.valtio.dev` or `http://valtiio.dev/zh` and then we'll need to configure the dns. Let's not change this yet until we get that sorted first though.

Reply:
> This link will need to change to something like `http://zh.valtio.dev` or `http://valtiio.dev/zh` and then we'll need to configure the dns. Let's not change this yet until we get that sorted first though.

@overthemike 
My opinion is no. We don't "host" non-English site, and it's not our responsibility. So, I prefer an external URL.